package.json

```diff
@@ -13,10 +13,10 @@
     "es"
   ],
   "jsnext:main": "./es/index.js",
-  "main": "./es/index.js",
+  "main": "./lib/index.js",
   "module": "./es/index.js",
   "scripts": {
-    "build": "rm -rf es && tsc",
+    "build": "rm -rf es && tsc && tsc -p ./tsconfig.es5.json",
```

Comment:
Alter the build completes, what will en up in the dist? The ESNext version, or the ES5 version? If both are you sure they don't acidentally overwrite each other?